AspNetCore-EchoBot-With-State/wwwroot/default.htm

```diff
@@ -0,0 +1,12 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+    <meta charset="utf-8" />
+</head>
+<body style="font-family:'Segoe UI'">
+    <h1>EchoBot sample using ASP.Net Core 2</h1>
+    <p>Describe your bot here and your terms of use etc.</p>
+    <p>Visit <a href="https://www.botframework.com/">Bot Framework</a> to register your bot. When you register it, remember to set your bot's endpoint to <pre>https://<i>your_bots_hostname</i>/api/messages</pre></p>
```

Comment:
Should you maybe sprinkle a little VanillaJS on this to dynamically generate the exact URL (w/port and whatnot) so people can copy/paste the exact value?

This is one of the top pieces of feedback we've consistently received just for the F5 experience with the emulator.

Reply:
Good idea. 

The new Emulator is adding a protocol handler, which will also be good.